applications/esl/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _esl_application:
+
+Electronic Shelf Label
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Electronic Shelf Label application provides both Access Point and Tag role.
```

Comment:
Could you add some more context to what kind of objective is this application trying to achieve?

Reply:
Add more content, please review it.